### ISSUE TITLE
Missing Imports for pgtype.Array with netip types

### ISSUE THREAD

**Issue by rhyselsmore** [kyleconroy/sqlc#2079]
### Version

1.16.0

### What happened?

Version: 1.17.0

It appears that arrays of `netip.Addr`/`netip.Prefix` do not add the relevant imports on generated files.

I'll dig into this shortly, just wanted to add it as a bug.

### Relevant log output

```shell
// Code generated by sqlc. DO NOT EDIT.
// versions:
//   sqlc v1.17.0
// source: route_update.sql

package db

import (
	"context"

	"github.com/jackc/pgx/v5/pgtype"
)

const createRouteUpdate = `-- name: CreateRouteUpdate :one
INSERT INTO route_updates (shard_ips)
  VALUES ($1::inet[])
  RETURNING id, shard_ips, created_at
`

type CreateRouteUpdateParams struct {
	ShardIps pgtype.Array[netip.Addr]
}

func (q *Queries) CreateRouteUpdate(ctx context.Context, db DBTX, arg CreateRouteUpdateParams) (RouteUpdate, error) {
	row := db.QueryRow(ctx, createRouteUpdate, arg.ShardIps)
	var i RouteUpdate
	err := row.Scan(
		&i.ID,
		&i.ShardIps,
		&i.CreatedAt,
	)
	return i, err
}
```

### Database schema

```sql
-- Route Updates
CREATE TABLE route_updates (
  id              bigserial NOT NULL,
  shard_ips       inet[] NOT NULL,
  created_at      timestamptz NOT NULL DEFAULT now()
);
```

### SQL queries

```sql
-- name: CreateRouteUpdate :one
INSERT INTO route_updates (shard_ips)
  VALUES (@shard_ips::inet[])
  RETURNING *;
```

### Configuration

```yaml
version: 1
packages:
  - path: "../"
    name: "db"
    engine: "postgresql"
    sql_package: "pgx/v5"
    schema: "./migration"
    emit_methods_with_db_argument: true
    queries:
      - "queries"
```

### Playground URL

Playground does not support v1.17.0

### What operating system are you using?

macOS

### What database engines are you using?

PostgreSQL

### What type of code are you generating?

Go

**Comment by zaneli** [comment]
Seems to fixed by #2090 ?
I try using latest revision 732c48e5e82fc3c9e6e1e21e5306e2d678d30279.

```go
// Code generated by sqlc. DO NOT EDIT.
// versions:
//   sqlc v1.17.0
// source: query.sql

package db

import (
	"context"
	"net/netip"
)

const createRouteUpdate = `-- name: CreateRouteUpdate :one
INSERT INTO route_updates (shard_ips)
  VALUES ($1::inet[])
  RETURNING id, shard_ips, created_at
`

func (q *Queries) CreateRouteUpdate(ctx context.Context, db DBTX, shardIps []netip.Addr) (RouteUpdate, error) {
	row := db.QueryRow(ctx, createRouteUpdate, shardIps)
	var i RouteUpdate
	err := row.Scan(&i.ID, &i.ShardIps, &i.CreatedAt)
	return i, err
}

```

**Comment by kyleconroy** [comment]
@zaneli thanks for the comment, I was going to write the same thing. This should be fixed in 1.17.2, which was just released.
